docs/07Examples/19-EEGProjects/20-Motor_Imagery.md

```diff
@@ -0,0 +1,152 @@
+
+Replace the Import Set with Import XDF
+
+Enter the calibration data filename
+Fill in the appropriate filename of the XDF file in the window.
+
+## Picking up Marker Streams with LSL
+
+![LSL Input](assets/TutorialImages/LSL_Input.png)
+EEG data LSL input
+
+The LSL Input node is responsible for returning a marker stream together with the EEG. Enter the name of the OpenBCI stream in the query and after you import the .xdf calibration data, you are ready to go.
+
+## Streaming the Data
+![OSC Output](assets/TutorialImages/OSC_Output.png)
+OSC output node to stream data
+
+Connect an OSC (Open sound control) Output node to the Logistic Regression node in the pipeline designer and configure it as shown below before you stream the data.
+
+![OSC Number_Output](assets/TutorialImages/OSC_Number_Output.png)
+
+OSC(Open Sound Control) output
+Type in the IP address of the device to which you want to stream the data, which can be either an Arduino or a Raspberry Pi). Use 127.0.0.1 as an IP address if you want to receive the data on your local computer.
+[Here](https://github.com/OpenBCI/OpenBCI_Tutorials/tree/master/Motor_Imagery) is a python code to receive the streamed data on the local computer.
+
+
+## Running the NeuroPype pipeline
+![NeuroPype Icon](assets/TutorialImages/Neuropype_Icon.png)
+
+NeuroPype icon in the taskbar
+
+We are in the final stage of the Motor Imagery Classification pipeline design. Now right click on the NeuroPype icon in the taskbar and click run pipeline. Navigate to your file path and select your edited pipeline simplemotorimagery.pyp and run it. If everything is configured properly, you will get two windows showing the Classification and Misclassification Rate. You can now see real-time predictions of either left or right on the windows. Imagine moving your right arm to increase the amplitude power of the right prediction and imagine moving your left arm to increase the amplitude power of the left prediction.
+
+![Classification Icon](assets/TutorialImages/Classification.png)
+
+When you run the python script on your local computer, you should receive the prediction data as shown below.
+
+![OSC output from python script](assets/TutorialImages/OSC_Python_output.png)
+
+OSC output from python script
+
+The car above uses NodeMCU and L298N motor driver. The NodeMCU coded in [Arduino IDE](https://www.arduino.cc/en/software) and the [code](https://github.com/OpenBCI/OpenBCI_Tutorials/tree/master/Motor_Imagery) is mentioned below. To learn how to use NodeMCU click [here](https://create.arduino.cc/projecthub/auggujarat/getting-started-with-nodemcu-esp8266-on-arduino-ide-b7e18e).
+
+
+This OSC protocol is widely used in fields like musical expression, robotics, video performance interfaces, distributed music systems, and inter-process communication. You can use it to drive motors, activate devices, and much more.This example video uses the OSC output to control the direction of rotation of the car.
```

Comment:
Before: "and much more.This example"
After change: "and much more. This example"